app/serializers/user_serializer.rb

```diff
@@ -1,4 +1,9 @@
 class UserSerializer < ActiveModel::Serializer
   attributes :id, :email, :zip, :first_name, :last_name,
              :slack_name, :mentor, :verified
+  attributes :role
```

Comment:
Good question.  With `ActiveModel::Serializer`, if the attribute is listed in the long list of attributes, it will directly call the method from the model.  In this case, `role` is an association`.  

So if it is left in the long list, it would return an object of the whole associated role, in the `role` key.

In this case, we just want the associated role's `title`.  To do that, you create a method with the desired name (i.e.) role, and have that method return whatever you want (i.e. just the `title`).